src/nomadic/start/commands.py

```diff
@@ -0,0 +1,108 @@
+import enum
+import os
+from contextlib import chdir
+from importlib.resources import files
+
+import click
+
+from nomadic.download.main import main as download_reference
+from nomadic.download.references import REFERENCE_COLLECTION
+from nomadic.util.config import default_config_path, write_config
+from nomadic.util.workspace import Workspace, init_workspace
+
+
+class Organism(enum.Enum):
+    pfalciparum = enum.auto()
+
+
+@click.argument(
+    "organism",
+    type=click.Choice(Organism),
+    required=True,
+)
+@click.option(
+    "-w",
+    "--workspace",
+    "workspace_path",
+    default="nomadic",
+    type=click.Path(exists=False),
+    help="Path of the the workspace where all the files will be stored",
+)
+@click.option(
+    "-r",
+    "--reference_name",
+    type=click.Choice(REFERENCE_COLLECTION),
+    help="Override the reference genome to be used in real-time analysis.",
+)
+@click.command(short_help="Help setting up nomadic.")
+def start(organism, workspace_path, reference_name) -> None:
+    """
+    Get started with nomadic.
+
+    This command will help you set up a new workspace for nomadic for a specific organism.
+    """
+
+    click.echo(f"Workspace will be created at: {workspace_path}")
+    init_workspace(workspace_path)
+    workspace = Workspace(workspace_path)
+
+    if organism == Organism.pfalciparum:
+        setup_pfalciparum(workspace, reference_name=reference_name)
+
+        click.echo(
+            "You can now enter your workspace with `cd nomadic` and run `nomadic realtime <experiment_name>` to start real-time analysis."
+        )
+
+
+def setup_pfalciparum(workspace, *, reference_name):
```

Comment:
For extensibility we could implement "setup_organism()" since I think it just needs a reference genome and list of BED files. We could have small class Organism(NamedTuple) or dataclasses that define the associated reference genome and bed files.

But for now this is totally fine of course

Reply:
It's hard to predict how similar other organisms will be. Maybe it will be very different, maybe it will be the same.
I would leave this to when we add more organism. I'ts not much work now, but we might have to change it back if it doesn't fit the next organism